tests/Form/InputResetTest.php

```
@@ -0,0 +1,476 @@
+        self::assertSame(
+            <<<HTML
+            <input id="inputreset-" type="reset" disabled>
+            HTML,
+            InputReset::tag()->id('inputreset-')->disabled(true)->render(),
+            "Failed asserting that element renders correctly with 'disabled' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(InputReset::class, ['class' => 'default-class']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)